pkg/ensurance/collector/types/types.go

```diff
@@ -43,14 +42,24 @@ const (
 )
 
 const (
-	NodeLocalCollectorType CollectType = "node-local"
+	NodeLocalCollectorType     CollectType = "node-local"
```

Comment:
types.go has special meaning in k8s, it define the api, maybe const.go is better than types.go?

Reply:
In k8s if it an type define is in types.go. if it only an const value, it in constants.go.